README.md

```diff
@@ -1,15 +1,15 @@
 [![Build Status](https://travis-ci.com/TRI-AMDD/beep.svg?branch=master)](https://travis-ci.com/TRI-AMDD/beep)
 [![Coverage Status](https://coveralls.io/repos/github/TRI-AMDD/beep/badge.svg?branch=master)](https://coveralls.io/github/TRI-AMDD/beep?branch=master)
 
-# Summary
+# BEEP
```

Comment:
This might be a better place to expand out BEEP to Battery Evaluation and Early Prediction.